app/models/friend_request.rb

```diff
@@ -1,7 +1,30 @@
+# frozen_string_literal: true
+
 class FriendRequest < ApplicationRecord
+  enum status: { pending: 'pending', accepted: 'accepted', declined: 'declined',
+                 cancelled: 'cancelled' }
+
   validates :sender, presence: true
```

Comment:
Pls see my annotations on `post.rb` and `comment.rb` related to validations for the same `belongs_to` attributes.